pkg/ctrlserver/graph/generated/generated.go

```diff
@@ -14,13 +14,12 @@ import (
 
 	"github.com/99designs/gqlgen/graphql"
 	"github.com/99designs/gqlgen/graphql/introspection"
+	"github.com/chaos-mesh/chaos-mesh/api/v1alpha1"
```

Comment:
Can you revert these changes?

Reply:
OK，i tried it，please wait a moment

Reply:
finished it ,i restored generate file changes